modules/core/src/v2/coins/fiat.ts

```diff
@@ -0,0 +1,76 @@
+/**
+ * @prettier
+ */
+import { BitGo } from '../../bitgo';
+import {
+  BaseCoin,
+  KeyPair,
+  ParsedTransaction,
+  ParseTransactionOptions,
+  SignedTransaction,
+  SignTransactionOptions,
+  VerifyAddressOptions,
+  VerifyTransactionOptions,
+} from '../baseCoin';
+import { MethodNotImplementedError } from '../../errors';
+
+export class Fiat extends BaseCoin {
```

Comment:
if we're not creating instances of this, can we make it `abstract` and remove the `createInstance()` method?

Reply:
Done!